doc/modules/ensemble.rst

```diff
@@ -1051,6 +1051,66 @@ multiplying the gradients (and the hessians) by the sample weights. Note that
 the binning stage (specifically the quantiles computation) does not take the
 weights into account.
 
+.. _categorical_support_gbdt:
+
+Categorical Features Support
+----------------------------
+
+:class:`HistGradientBoostingClassifier` and
+:class:`HistGradientBoostingRegressor` have native support for categorical
+features: they can consider splits on non-ordered, categorical data.
+
+For datasets with categorical features, using the native categorical support
+is often better than relying on one-hot encoding
+(:class:`~sklearn.preprocessing.OneHotEncoder`), because one-hot encoding
+requires more tree depth to achieve equivalent splits. It is also usually
+better to rely on the native support rather than to treat categorical
+features as continuous, since categories are nominal quantities where order
```

Comment:
```suggestion
features as ordinal, since categories are nominal quantities where order
```

Reply:
We don't imply OHE, the features are considered continuous / ordinal iff we don't use the native categorical suppport. I'll change continuous for "continuous/ordinal" and "native support" with "native categorical support" which will hopefully clarify things